### ISSUE TITLE
Slow texture loading

### ISSUE THREAD

**Issue by d-level** [matteokeole/raven#19]
When the texture list file contains ~40 entries the loading time is ~1s.

### Proposition

Use `Promise.all` to fetch the textures in parallel.  
Some textures can also use deferred loading if they are used later.

**Comment by d-level** [comment]
Added parallel texture loading with `Promise.allSettled`.
